admin/package.json

```diff
@@ -5,9 +5,7 @@
   "version": "1.0.0",
   "private": false,
   "scripts": {
-    "start": "run-script-os",
-    "start:windows": "SET PORT=3030 && react-scripts start",
-    "start:all": "PORT=3030 react-scripts start",
+    "start": "PORT=3030 react-scripts start",
```

Comment:
you completely overrode my changes here. you have to take care when dealing with merge conflicts, as it's not something that you just simply accept one or the other. in this case here it's very clear that my change is the one that makes sense (as I added support for windows with the run-script-os package, and you just completely deleted my change by not caring about it. luckily it's a very soft change and I can just do it again, but keep in mind this is not the way to go and you should always watch carefuly when solving merge conflicts.